src/contracts/definitions/derivativeShd.ts

```diff
@@ -0,0 +1,115 @@
+import { snip20 } from '~/contracts/definitions/snip20';
+import { generatePadding } from '~/lib/utils';
+
+/**
+ * Query the public contract staking info
+ */
+const msgQueryStakingInfo = () => ({ staking_info: {} });
+
+/**
+ * Query the fee info
+ */
+const msgQueryFeeInfo = () => ({ fee_info: {} });
```

Comment:
i think we can remove this because everything is built into the info query.

Reply:
Done